models/Item.model.js

```diff
@@ -0,0 +1,16 @@
+const { Schema, model } = require('mongoose');
+
+const itemSchema = new Schema(
+  {
+    name: {
+      type: String,
+    },
+    done: Boolean,
+    deadline: String,
```

Comment:
I think you can save Dates in Mongoose :)

Reply:
https://mongoosejs.com/docs/tutorials/dates.html

Reply:
very useful stuff :+1:

Reply:
you can even set a default to be "now" + say 1 day or something like that :blush:

Reply:
At first we tried to use mongoose type Date, but it clashed with the date input, because the two of them are formatted differently. So I'm sure it's possible to convert those formats between input and db, but it turned out to be much simpler to use mongoose type string, because this way the date could just stay in the format of the date input